### PR TITLE
Playwright examples with package.json

### PR DESCRIPTION
**Pull request by nadvolod**
* adding a `package.json` to the project to make this more realistic to how customers will use this
* added some default test examples that came with playwright
* Removed an example test that we need to maintain. Instead, I left a default one that Playwright provides. Less maintenance for us 👍  Not sure how you all feel about that?

Some noteworthy behaviors
`npm run test.local` will pass
`npm run test.saucectl` will fail

### CODE REVIEW

File: .sauce/config.yml [hunk 1]
```diff
@@ -25,48 +26,21 @@ suites:
     testMatch: ['.*.js']
     params:
       browserName: "firefox"
+      project: "firefox"  #an important key to override what's in the config.js
```
Comment (alexplischke) [inline .sauce/config.yml:29]:
```suggestion
      project: "firefox"  # Runs the project that's defined in `playwright.config.js`
```

File: .sauce/config.yml [hunk 2]
```diff
@@ -10,12 +10,13 @@ sauce:
       - e2e
       - release team
       - other tag
-    build: $CI_COMMIT_SHORT_SHA
+    build: "config-$CURRENT_TIME"
```
Comment (alexplischke) [inline .sauce/config.yml:13]: `$CURRENT_TIME` does not exist in our environment. So this won't really do for our pipelines here.

Btw. not sure if you knew, but if you leave out `build` entirely, saucectl will automatically generate a build with a timestamp. So the above is entirely unnecessary.

Comment (nadvolod) [reply .sauce/config.yml:13]: @alexplischke had no idea! very cool. Let me try

Comment (nadvolod) [reply .sauce/config.yml]: @alexplischke excellent functionality! Fixed

**Comment by nadvolod** [comment]
@alexplischke if you approve this, can you kindly merge?

**Comment by nadvolod** [comment]
@alexplischke @saucelabs/devx any updates here?

**Review by alexplischke — APPROVED** [review]

**Pull request merged**